### PR TITLE
- fixed empty '{}' format ( "{}".format doesn't work on python < 2.7 )

### PR DESCRIPTION
**Pull request by midnightercz**

### CODE REVIEW

File: plugins/pulp_docker/plugins/importers/sync.py [hunk 1]
```diff
@@ -306,5 +306,5 @@ def _move_file(self, unit):
         :param unit: a pulp unit
         :type  unit: pulp.plugins.model.Unit
         """
-        _logger.debug('moving files in to place for Unit {}'.format(unit))
```
Comment (seandst) [inline plugins/pulp_docker/plugins/importers/sync.py]: `'{}'.format` may not work, but reverting to percent interpolation is a bit harsh. how about `'{0}'.format`?

File: plugins/pulp_docker/plugins/importers/sync.py [hunk 2]
```diff
@@ -306,5 +306,5 @@ def _move_file(self, unit):
...
+        _logger.debug('moving files in to place for Unit %s' % unit)
```
Comment (rbarlow) [inline plugins/pulp_docker/plugins/importers/sync.py]: I agree with @seandst: let's change this line to:

```
_logger.debug('moving files in to place for Unit {unit}'.format(unit=unit))
```

Let me know when you've made this change and I'll merge.

**Comment by seandst** [comment]
ok test

File: plugins/pulp_docker/plugins/importers/sync.py [hunk 3]
```diff
@@ -306,5 +306,5 @@ def _move_file(self, unit):
         :param unit: a pulp unit
         :type  unit: pulp.plugins.model.Unit
         """
-        _logger.debug('moving files in to place for Unit {}'.format(unit))
+        _logger.debug('moving files in to place for Unit {unit}' % format(unit=unit))
```
Comment (seandst) [inline plugins/pulp_docker/plugins/importers/sync.py]: This is confusing the two interpolation methods. There should be no percent signs.

```python
_logger.debug('moving files in to place for Unit {unit}'.format(unit=unit))
```

**Comment by rbarlow** [comment]
Hello @midnightercz,

It's been about a month since the last comment on this PR. Do you think you can make the change that @seandst requested?

**Pull request merged**